tests/models_tests/models_test.go

```diff
@@ -1037,3 +1038,25 @@ func TestPersonConceptAndCountString(t *testing.T) {
 	}
 
 }
+
+func TestGetDataDictionaryFail(t *testing.T) {
+	setUp(t)
+	data, _ := dataDictionaryModel.GetDataDictionary()
+
+	if data != nil {
```

Comment:
what if the `TestGenerateDataDictionary` runs first? Will this one fail? Do we know if the tests are executed in the order they are found in this test file?

Reply:
maybe one idea is to just combine this and the `TestGenerateDataDictionary` test methods into a single one?